### DISCUSSION TITLE
Failsafe RTH with no compass/magetometer in acro mode (multicopter) : Does it work ?

### DISCUSSION THREAD

**Discussion by alccrl** [iNavFlight/inav#9859] [Q&A]
Hi.

Documentation (failsafe.md) says : 

RTH: (Return To Home) One of the key features of INAV, it automatically navigates the craft back to the home position and (optionally) lands it. Similarly to all other automated navigation methods, it requires GPS for any type of craft, plus compass and barometer for multicopters.

I had a look at issues, and I do not understand if actually RTH with no compass/magnetomer works in inav 7.0.1, beacuse there is some activity about it (failsafe.md may be not updated ??).

Does anybody know ?

Alfredo

**Comment by sensei-hacker** [comment]
It is anticipated that in a version to be released very soon, it can *try* to RTH despite a missing compass. This can work well if the pilot flew straight and true for a while shortly before the RTH event, if the wind isn't gusty or changing.

So the two main points are:
Not yet, but 7.1 is coming soon

It doesn't mean you shouldn't have a compass

Kinda like walking around your house with your eyes open vs closed. It may work out perfectly fine with your eyes closed.  You may do perfectly. Or not. Opening your eyes gives you a better chance, and having a compass is similar.

Thanks for pointing out that page in the documentation should be updated to point to the new page about no-compass operation!
